index.js

```diff
@@ -1,51 +1,111 @@
 'use strict';
 const alfy = require('alfy');
+const utils = require('./js/utilities');
+const formatting = require('./js/formatting');
+const baseUrl = 'https://api.darksky.net/forecast/';
+const excludes = '?exclude=minutely,hourly&units=auto';
 
-//fetches the weather for Valencia
-alfy.fetch('https://api.darksky.net/forecast/b3bfdcda400630b909f112b1530aea93/37.8267,-122.4233', {
-    query: {
-        q: alfy.input
+//fetches the weather, based on input or default settings
+try {
+    
+    var mapsAPIKey = alfy.cache.get('Google Maps API Token');
+    var locationQuery;
+
+    if (alfy.input)
+    {
+        locationQuery = utils.parseLocationQuery(alfy.input);
+    } else {
+        locationQuery = utils.parseLocationQuery(utils.getConfiguration('Default Location'));
     }
-}).then(data => {
-    let dayName = '';
-    let displayTemperature = '';
-
-	const items = data.results
-    .map(x => {
-        const date = new Date(x.time*1000); //API time format is UNIX ... needs to be *1000 to be ticks
-        const high = x.temperatureHigh;
-        const low = x.temperatureLow;
-        
 
-        dayName += date.getDayName();
-        displayTemperature += getFormattedTemperatures(high, low);
+    if (mapsAPIKey){
+        //Use google maps to determine location and get weather
+        getWeatherThroughGoogleMaps(locationQuery);
+    } else if (locationQuery.type == 'latlng'){
```

Comment:
You should be using `=== `, I won't go into why, but a linter with the Airbnb styleguide would have caught this as well.